BUILD.md

```diff
@@ -2,61 +2,90 @@
 
 ## From source
 
-1. On Red Hat Enterprise Linux and Fedora make sure the packages `openscap-utils`, `openscap-python`, and `python-lxml` and their dependencies are installed. We require version `1.0.8` or later of `openscap-utils` (available in Red Hat Enterprise Linux) as well as `git`. 
+1. On Red Hat Enterprise Linux and Fedora make sure the packages `openscap-utils`, `openscap-python`, and `python-lxml` and their dependencies are installed. We require version `1.0.8` or later of `openscap-utils` (available in Red Hat Enterprise Linux) as well as `git`.
 
- `# yum -y install git openscap-utils openscap-python python-lxml`
+ `# yum -y install git cmake openscap-utils openscap-python python-lxml`
```

Comment:
Add a reference to cmake in the prose, e.g. "..... make sure the packages **cmake**, openscap-utils, openscap-python...."